src/main/java/net/sf/jabref/gui/EntryTypeDialog.java

```diff
@@ -91,12 +112,17 @@ private JPanel createEntryGroupsPanel() {
         JPanel panel = new JPanel();
         panel.setLayout(new VerticalLayout());
 
-        if(biblatexMode) {
+        if (biblatexMode) {
             panel.add(createEntryGroupPanel("BibLateX", EntryTypes.getAllValues(bibDatabaseContext.getMode())));
         } else {
             panel.add(createEntryGroupPanel("BibTeX", BibtexEntryTypes.ALL));
             panel.add(createEntryGroupPanel("IEEETran", IEEETranEntryTypes.ALL));
-            panel.add(createEntryGroupPanel(Localization.lang("Custom"), CustomEntryTypesManager.ALL));
+
+            if (!CustomEntryTypesManager.ALL.isEmpty()) {
```

Comment:
No, they are already displayed (even in biblatex mode)  but they are mixed with the others:

![jabrefmycustomentrytype](https://cloud.githubusercontent.com/assets/320228/18309431/ad6d4b02-74f9-11e6-9b22-bac92b39076d.png)